src/core/src/param.rs

```diff
@@ -33,12 +33,43 @@ pub unsafe extern "C" fn param_close(param: *mut Param) {
     drop(Box::from_raw(param));
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn param_app_ver_get(param: &Param, buf: &mut QString) {
+    match &param.app_ver() {
+        Some(app_ver) => buf.set(app_ver),
+        None => buf.set("No App_Ver"),
+    }
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn param_category_get(param: &Param, buf: &mut QString) {
+    buf.set(param.category());
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn param_contentId_get(param: &Param, buf: &mut QString) {
+    buf.set(param.content_id());
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn param_shortContentId_get(param: &Param, buf: &mut QString) {
+    buf.set(param.shortcontent_id());
+}
+
 #[no_mangle]
 pub unsafe extern "C" fn param_title_get(param: &Param, buf: &mut QString) {
-    buf.set(param.title());
+    match &param.title() {
+        Some(title) => buf.set(title),
+        None => buf.set("No Title"),
```

Comment:
Better to use an empty string and let the top-level user decide what to do instead.